fr/_posts/2024-06-10-boite-outils-durgence.md

```diff
@@ -0,0 +1,100 @@
+<li><a href="https://blogue.canada.ca/2020/07/15/fatigue-dalerte.html">La « fatigue d’alerte » pendant la COVID-19</a></li>
+</ul>
+
+<h2>Fournir des réponses et non des renseignements</h2>
+<ul>
+<li>En situation de crise, les gens sont accablés et font face à une avalanche de nouvelles informations.</li>
+<li>Au moment de chercher des réponses à leurs questions sur Canada.ca, ils doivent avoir la certitude d’avoir trouvé une réponse complète et adaptée à leur tâche ou situation.</li>
+<li>Le contenu interactif permet aux utilisateurs et utilisatrices de trouver plus rapidement des réponses et les guide vers les renseignements dont ils et elles ont besoin pour réaliser une tâche. Il leur évite ainsi de chercher des informations dans une multitude de pages.</li>
+<li>Il faut se préparer à coordonner les différentes ressources provinciales et à fournir des liens vers ces dernières. Voici un exemple de la façon dont cela a été réalisé pour le contenu relatif à la vaccination contre la COVID-19&nbsp;:
+<ul>
+ <li><a href="https://www.canada.ca/fr/sante-publique/services/maladies/maladie-coronavirus-covid-19/vaccins/comment-vacciner.html#a1">Manière de se faire vacciner&nbsp;: Prendre un rendez-vous pour recevoir votre vaccin</a></li>
+ </ul></li>
+</ul>
+<p>Pour en savoir plus sur la manière de remplacer les renseignements par des réponses, consultez les pages suivantes&nbsp;:</p>
+<ul>
+<li><a href="https://conception.canada.ca/resumes-recherche/etude-sur-les-restrictions-de-voyage.html">Utilisation d’un assistant afin d’aider les gens à se renseigner durant la pandémie de la COVID-19</a></li>
+<li><a href="https://blogue.canada.ca/2020/11/12/listes-de-verification-interactives.html">Utilisation de listes de vérification interactives pour simplifier les critères d’admissibilité</a></li>
+<li><a href="https://blogue.canada.ca/2021/04/08/utilisation-de-questions-interactives.html">Les questions interactives pour aider les gens</a></li>
+</ul>
+
+<h2>Créer une structure permettant aux gens de trouver du contenu</h2>
+<ul>
+<li>Un contenu peut être disponible sans pour autant être facile à trouver ou à lire.</li>
+<li>L’utilisation de termes courants et la simplification des pages de navigation afin de faciliter le balayage visuel permettent aux utilisateurs et utilisatrices de trouver rapidement les renseignements recherchés.</li>
+<li>En temps de crise, les gens sont plus susceptibles de visiter les sites Web sur leur téléphone. Il faut donc concevoir le contenu en vue d’une navigation sur mobile plutôt que de privilégier la version pour ordinateur.</li>
+</ul>
+
+<p>Pour plus de renseignements sur la création d’une structure permettant aux gens de trouver du contenu, consultez les pages suivantes&nbsp;:</p>
+<ul>
+<li>Des pages de navigation flexibles pour les situations d’urgence <a href="https://docs.google.com/document/d/1YF6qGdgjzFrKVdnXBD_wRp_29tUyF4TNOE3w50opTKE/edit#heading=h.gjdgxs">(brouillon)</a></li>
```

Comment:
Please remove: (brouillon)

Please instead link to the blog post, the link is: https://blogue.canada.ca/2024/05/13/conception-de-page-de-navigation-flexible-pour-les-urgences

Reply:
Fixed